app/com/linkedin/drelephant/spark/heuristics/ConfigurationHeuristic.scala

```diff
@@ -77,6 +74,14 @@ class ConfigurationHeuristic(private val heuristicConfigurationData: HeuristicCo
       new HeuristicResultDetails(
```

Comment:
I don't understand why do we need to have an if statement here, we don't have it in any other case. I am already adding details like the following:

new HeuristicResultDetails(
        SPARK_DYNAMIC_ALLOCATION_ENABLED,
        formatProperty(evaluator.isDynamicAllocationEnabled.map(_.toString))
      ),
      new HeuristicResultDetails(
        SPARK_SHUFFLE_SERVICE_ENABLED,
        formatProperty(evaluator.isShuffleServiceEnabled.map(_.toString))
      )